metadata_backend/api/handlers.py

```diff
@@ -164,14 +164,20 @@ async def validate(self, req: Request) -> Response:
             raise web.HTTPBadRequest(reason=reason)
 
         except ParseError as error:
-            reason = f"Faulty XML file was given. Details: {error}"
-            body = json.dumps({"isValid": False, "reason": reason})
+            detail = f"Faulty XML file was given.\nERROR: {error}"
+            body = json.dumps({"isValid": False, "detail": detail})
             return web.Response(body=body,
                                 content_type="application/json")
 
         except XMLSchemaValidationError as error:
-            reason = f"XML file is not valid. Details: {error}"
-            body = json.dumps({"isValid": False, "reason": reason})
+            '''
+            # Parsing the validation error message for reason and instance
+            from xml.etree import ElementTree
+            reason = error.reason
+            instance = ElementTree.tostring(error.elem, encoding="unicode")
+            '''
+            detail = f"XML file is not valid against schema.\nERROR: {error}"
```

Comment:
```suggestion
            detail = f"XML file is not valid against schema.\nERROR: {error.reason}"
```

Reply:
Yea, found that as well. The element where it's located (instance = `ElementTree.tostring(error.elem, encoding="unicode")`) might also be useful info to include

Reply:
yeap. :+1: